Lib/tkinter/__init__.py

```diff
@@ -1527,10 +1527,21 @@ def bind(self, sequence=None, func=None, add=None):
         return self._bind(('bind', self._w), sequence, func, add)
 
     def unbind(self, sequence, funcid=None):
-        """Unbind for this widget for event SEQUENCE  the
-        function identified with FUNCID."""
-        self.tk.call('bind', self._w, sequence, '')
-        if funcid:
+        """Unbind for this widget the event SEQUENCE.
+
+        If FUNCID is given, only unbind the function identified with FUNCID
+        and also delete that command.
```

Comment:
Make clearer that this will not delete the Python function passed to the binding call, but only the binding-specific (tcl) wrapper (as the bind docstring promises).
```suggestion
        and also delete the binding-specific tcl wrapper function.
```

Reply:
What is "the binding-specific tcl wrapper function"? This term is not mentioned in the documentation and will confuse users.

It deletes the Tcl command and removes a reference to Python function. These details are not documented, but experienced users can discover this. I do not know what to write here.

Reply:
Yes, it does not work with keyboard events.

I used different events for different tests to avoid possible order dependencies or chain reaction of failures after failing in one test. It was easy with keyboard events, it also minimizes risk of generating even by a user during running tests. But it may be not so easy to find a dozen different non-keyboard events (and we can add more tests in future). Is it necessary?

Reply:
I withdraw the suggestion above.  Either insert 'tcl' before 'command' or wait until someone reports being confused.

Reply:
Done.